src/main/java/Duke.java

```diff
@@ -1,10 +1,103 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASK = 100;
+    public static final String lineSplit = "    ____________________________________________________________";
+
     public static void main(String[] args) {
+        boolean canExit = false;
+        String line;
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[MAX_TASK];
+        int taskAmount = 0;
+
+        //the logo and greet
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greet();
+
+        //main chat box
+        while(!canExit) {
+            line = in.nextLine();
+            if(line.equals("list")) {
+                listTasks(taskAmount, tasks);
+            }
+            else if(line.equals("bye")) {
+                canExit = true;
+            }
+            else if(line.contains("done")) {
+                markAsDone(line, tasks, taskAmount);
+            }
+            else {
+                if(line.contains("todo")) {
+                    tasks[taskAmount] = new Todo(line);
+                }
+                else if(line.contains("deadline")) {
+                    tasks[taskAmount] = new Deadline(line);
+                }
+                else if(line.contains("event")) {
+                    tasks[taskAmount] = new Event(line);
+                }
+                else {
+                    tasks[taskAmount] = new Task(line);
+                }
+                tasks[taskAmount].doneAddTask();
+                taskAmount++;
+            }
+        }
+        exit();
+    }
+
+    public static void greet() {
+        System.out.println(lineSplit);
+        System.out.println("    Hello! I'm Duke");
+        System.out.println("    What can I do for you?");
+        System.out.println(lineSplit);
+    }
+
+    public static void echo(String message) {
+        System.out.println(lineSplit);
+        System.out.println("    " + message);
```

Comment:
I noticed the same 4 space strings are used to indent your output in the terminal in several other places too. Perhaps you can consider introducing a constant to enhance readability and ease future code maintenance.